chore(scripts/RuinsOfAhnQiraj): cleanup ruins_of_ahnqiraj.h

Si1ker opened the pull request:
<!-- First of all, THANK YOU for your contribution. -->

## Changes Proposed:
- Remove unused code
- Remove duplicates

<!-- If you intend to contribute repeatedly to our project, it is a good idea to join our discord channel. We set ranks for our contributors and give them access to special resources or knowledge: https://discord.com/invite/DasJqPba)
     Do not remove the instructions below about testing, they will help users to test your PR -->
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

Maelthyrr commented:
Need to be update since the update to the new script system.

Si1ker commented:
> Need to be update since the update to the new script system.

@Maelthyrr Done.

Si1ker commented:
@Maelthyrr labels?
@IntelligentQuantum approve?

Nyeriah commented on src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp:
```diff
@@ -180,8 +180,7 @@ struct boss_ayamiss : public BossAI
                     instance->SetGuidData(DATA_PARALYZED, target->GetGUID());
-                    uint8 index = urand(0, 1);
-                    me->SummonCreature(NPC_HIVEZARA_LARVA, LarvaPos[index], TEMPSUMMON_TIMED_OR_CORPSE_DESPAWN, 30000);
+                    DoCastAOE(RAND(SPELL_SUMMON_LARVA_A, SPELL_SUMMON_LARVA_B), true);
```
Something seems to have happened here, merge conflict probably went wrong, not sure if it is just me

![image](https://user-images.githubusercontent.com/47818697/184444744-1274d8f8-a992-4558-ab32-797c89434577.png)

Nyeriah commented on src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp:
```diff
@@ -201,8 +200,13 @@ struct boss_ayamiss : public BossAI
+                if (Creature* wasp = me->SummonCreature(NPC_HIVEZARA_SWARMER, spawnpos, TEMPSUMMON_CORPSE_TIMED_DESPAWN, 5000))
+                {
+                    wasp->GetMotionMaster()->MoveRandom(10.0f);
+                }
```
here too, event will never repeat, merge conflict def went wrong

Si1ker commented:
@Nyeriah ?????

any complaints about what's already in the core go for your pr on ayamiss.
I just removed unused stuff and renamed it to (old, first AC commit ever) already existing values.

Nyeriah commented:
I posted images of what I saw, and the merge had issues

I see that those issues were either corrected already or GitHub had issues displaying the merge PR

I appreciate your willingness to help, but unfurtunately I see there's more effort spent on snarky remarks than addressing the concerns and questions raised by whoever reviews your PRs
